src/Plugins/ComplexCore/src/ComplexCore/Filters/ImportCSVDataFilter.cpp

```diff
@@ -343,88 +362,213 @@ IFilter::UniquePointer ImportCSVDataFilter::clone() const
   return std::make_unique<ImportCSVDataFilter>();
 }
 
+//------------------------------------------------------------------------------
+IFilter::PreflightResult ImportCSVDataFilter::readHeaders(const std::string& inputFilePath, usize headersLineNum, bool useTab, bool useSemicolon, bool useSpace, bool useComma,
```

Comment:
This function could take the header cache as an argument instead of being a member function.